ganalytics.d.ts

```diff
@@ -0,0 +1,53 @@
+interface IGAnalyticsOptions {
+
+    /**
+     * Anonymize the sender's IP address.
+     * See [Anonymize IP](https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#aip).
+     */
+    aip?: number;
+
+    /**
+     * Specifies the application's name.
+     * See [Application Name](https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#an).
+     */
+    an?: string;
+
+    /**
+     * Specifies the application identifier.
+     * See [Application ID](https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#aid).
+     */
+    aid?: string;
+
+    /**
+     * Specifies the application installer identifier.
+     * See [Installer ID](https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#aiid).
+     */
+    aiid?: string;
+
+    /**
+     * Specifies the application verison.
+     * See [Version](https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#av).
+     */
+    av?: string;
+
+    /**
+     * Indicates the data source type of the hit; eg web or app.
+     * See [Data Source](https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#ds).
+     */
+    ds?: string;
+}
+
+interface IGAnalytics {
+    send: (kind: "pageview" | "event", params?: any) => void;
```

Comment:
There's actually a lot more _kinds_ that can be sent. Here's the list: https://github.com/lukeed/ganalytics#type

So let's add a new Enum here: 

```ts
send: (kind: EventType, params?: object) => void
```

> Also, `params` can only be an object.